### PR TITLE
v1.16 stable branch preparation

### PR DESCRIPTION
**Pull request by aanm**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -70,11 +70,14 @@
 #   Provide recommendations about the types, names and labels for metrics to
 #   follow best practices. This includes considering the cardinality impact of
 #   metrics being added or extended.
-# - @cilium/release-managers:
-#   Review files related to releases like AUTHORS and VERSION.
```
Comment (joestringer) [inline CODEOWNERS]: No need to delete these lines.

File: install/kubernetes/Makefile [hunk 2]
```diff
@@ -47,7 +47,7 @@ update-chart: $(ROOT_DIR)/VERSION # Update chart versions to point to the curren
 	$(ECHO_GEN)cilium/Chart.yaml
 	$(QUIET)grep -lR -e 'version:' -e 'appVersion:' $(CILIUM_CHARTS)/ \
 		| xargs -L 1 $(SED) -i -e 's/'$(CILIUM_CHART_REGEX)'/\1 '$(VERSION)'/g'
-	$(QUIET)$(SED) -i 's;icon:.*;icon: $(LOGO_BASE_URL)/cilium@main/$(LOGO_PATH);' $(CHART_FILE)
+	$(QUIET)$(SED) -i 's;icon:.*;icon: $(LOGO_BASE_URL)/cilium@$(CILIUM_BRANCH)/$(LOGO_PATH);' $(CHART_FILE)
```
Comment (joestringer) [inline install/kubernetes/Makefile]: Going forward I think we should just rely on the `main` path for this. See https://github.com/cilium/cilium/pull/32776 .

File: install/kubernetes/Makefile.values [hunk 3]
```diff
@@ -25,6 +28,10 @@ else
     export HUBBLE_RELAY_REPO:=${CI_REGISTRY}/${CI_ORG}/hubble-relay-ci
 endif
 
+ifndef CILIUM_BRANCH
+$(error "CILIUM_BRANCH needs to be defined")
+endif
+
```
Comment (joestringer) [inline install/kubernetes/Makefile.values]: Same thing here, we don't need to make this change. See https://github.com/cilium/cilium/pull/32776

File: .github/workflows/tests-ipsec-upgrade.yaml [hunk 4]
```diff
@@ -129,7 +126,7 @@ jobs:
 
           - config: 'bpf-next'
             # renovate: datasource=docker depName=quay.io/lvh-images/kind
-            kernel: 'bpf-next-20240625.013127'
+            kernel: '6.6-20240620.102250'
```
Comment (joestringer) [inline .github/workflows/tests-ipsec-upgrade.yaml:129]: I recognize this is making the CI results more stable, but it also gets me wondering what if a newer kernel breaks an older release, do we have any early warning system for that? Or we just rely on the kernel breaking the main branch and then figure out whether we need to apply mitigations back to older releases?

**Review by joestringer — COMMENTED** [review]
Couple of minor nits here - no need to re-add the CILIUM_BRANCH stuff, and the codeowners file shouldn't lose group descriptions.

**Review by joestringer — COMMENTED** [review]
A few more comparisons to https://github.com/cilium/cilium/pull/29838:

- We should remove the scheduled scale tests from this branch
- Remove stable.txt from the branch

We also re-ran `make -C install/kubernetes` and updated the CRD schema docs during the v1.15 stable branch prep, but I don't know if that matters so much, we'll do that as part of the next RC anyway.

File: install/kubernetes/Makefile.values (hunk 3 above)
Comment (aanm) [reply install/kubernetes/Makefile.values]: ohh finally. I didn't know about #32776

File: install/kubernetes/Makefile (hunk 2 above)
Comment (aanm) [reply install/kubernetes/Makefile]: ohh finally. I didn't know about #32776

File: .github/workflows/tests-ipsec-upgrade.yaml (hunk 4 above)
Comment (joestringer) [reply .github/workflows/tests-ipsec-upgrade.yaml:129]: One more minor interesting note here - 6.6 is the current LTS, but bpf-next is closer to 6.10. This is subtly changing the target tests. Though that said, over the lifetime of maintaining v1.16 it makes sense to test 6.6 (EOL ETA end of 2026)

Comment (aanm) [reply .github/workflows/tests-ipsec-upgrade.yaml:129]: I'll follow up next week with https://github.com/cilium/cilium/issues/33460

**Review by joestringer — APPROVED** [review]
:ship:

**Review by joestringer — APPROVED** [review]

**Pull request merged**